src/main/java/stsjorbsmod/cards/cull/OldBook.java

```diff
@@ -0,0 +1,59 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.cards.OnEntombedSubscriber;
+import stsjorbsmod.characters.Cull;
+import stsjorbsmod.patches.EntombedField;
+import stsjorbsmod.powers.OldBookPower;
+
+import static stsjorbsmod.JorbsMod.JorbsCardTags.LEGENDARY;
+
+public class OldBook extends CustomJorbsModCard implements OnEntombedSubscriber {
+    public static final String ID = JorbsMod.makeID(OldBook.class);
+
+    private static final CardRarity RARITY = CardRarity.RARE;
+    private static final CardTarget TARGET = CardTarget.SELF;
+    private static final CardType TYPE = CardType.SKILL;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private AbstractPlayer p = AbstractDungeon.player;
+
+    private static final int COST = COST_UNPLAYABLE;
+    private static final int HEAL_PERCENT = 0;
+    private static final int UPGRADE_HEAL_PERCENT = 10;
+
+    public OldBook() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        EntombedField.entombed.set(this, true);
+
+        magicNumber = baseMagicNumber = HEAL_PERCENT;
+
+        tags.add(LEGENDARY);
+    }
+
+    @Override
+    public void onCardEntombed() {
+        addToTop(new ApplyPowerAction(p, p, new OldBookPower(AbstractDungeon.player, this)));
```

Comment:
Should probably also have an `onCardExhumed` condition to reduce the power, to in case some other effect manages to exhume the card somehow (eg, the player somehow obtains a copy of the card Exhume).